telegram/ext/_chatjoinrequesthandler.py

```diff
@@ -43,17 +44,60 @@ async def callback(update: Update, context: CallbackContext)
 
             The return value of the callback is usually ignored except for the special case of
             :class:`telegram.ext.ConversationHandler`.
+        chat_id (SCT[:obj:`int`], optional): Filters requests to allow only
```

Comment:
I am sorry, I mislead you here. While the type hint for the argument is shortcutable, the sphinx docs don't work this way sadly. So you still have to do ```:obj:`int` | Collection[:obj:`int`]``` here.